Testscripts/Linux/verify_vmbus_interrupts.sh

```diff
@@ -45,28 +50,37 @@ function verify_vmbus_interrupts() {
     #
     # Verify if VMBUS interrupts are processed by all CPUs
     #
-    while read line
+    cat /proc/interrupts > interrupts
+    while read -r line
     do
-        if [[ ($line = *hyperv* ) || ( $line = *Hypervisor* ) ]]; then
-            for (( core=0; core<=$cpu_count-1; core++ ))
+        if [[ ($line = *Hyper-V* ) || ( $line = *Hypervisor* ) ]]; then
```

Comment:
I have a try. If we change this to "Hyper-V", the nonCPU0inter will be double compared with before.

Reply:
I don't understand your comment. Can you elaborate? In this case, mainly we track down "Hypervisor callback interrupts".

Reply:
If we  judge ($line = *Hyper-V* ), "Hyper-V stimer0 interrupts" is also tracked down. 
This test results "All {497} CPU cores are processing interrupts. 81 CPUs are processing multiple interrupts" you attached in the above comment window, 497 is 416 + 81, 416 this is from "Hyper-V stimer0 interrupts", and 81 is from "Hypervisor callback interrupts".

Reply:
Please double check. If my expression is also not clear enough, we can have a call.